clab/config.go

```diff
@@ -46,11 +46,23 @@ type Config struct {
 	Prefix   *string         `json:"prefix,omitempty"`
 	Mgmt     *types.MgmtNet  `json:"mgmt,omitempty"`
 	Topology *types.Topology `json:"topology,omitempty"`
+	Settings *Settings       `yaml:"settings"`
```

Comment:
This should be part of the Topology struct, and parsed via appropriate method

I agree with the name `settings`

Reply:
done

Reply:
I was dumb. This belongs at the root of the clab file. I moved it back, sorry.